tests/puppeteer/campaigns/functional/BO/catalog/monitoring/02_monitoringProducts.js

```diff
@@ -0,0 +1,195 @@
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductsPage = require('@pages/BO/catalog/products');
+const AddProductPage = require('@pages/BO/catalog/products/add');
+const MonitoringPage = require('@pages/BO/catalog/monitoring');
+const ProductFaker = require('@data/faker/product');
+
+let browser;
+let page;
+let numberOfProducts = 0;
+let numberOfProductsIngrid = 0;
+const productWithoutCombinations = new ProductFaker(
+  {type: 'Standard product', productHasCombinations: false},
+);
+const productWithoutCombinationsWithoutQuantity = new ProductFaker(
+  {type: 'Standard product', productHasCombinations: false, quantity: '0'},
+);
+const productWithCombinationsWithoutQuantity = new ProductFaker(
+  {type: 'Standard product', productHasCombinations: true, quantity: '0'},
+);
+const productWithoutPrice = new ProductFaker(
+  {type: 'Standard product', productHasCombinations: false, price: '0'},
+);
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    productsPage: new ProductsPage(page),
+    addProductPage: new AddProductPage(page),
+    monitoringPage: new MonitoringPage(page),
+  };
+};
+
+/*
+Create new product
+Check existence of new product in monitoring page
+Delete product and check deletion in products page
+ */
+describe('Create different products and delete them from monitoring page', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO
+  loginCommon.loginBO();
+
+  const tests = [
+    {
+      args: {
+        productType: 'without image',
+        productToCreate: productWithoutCombinations,
+        gridName: 'product_without_image',
+        enabled: true,
+      },
+    },
+    {
+      args: {
+        productType: 'disabled',
+        productToCreate: productWithoutCombinations,
+        gridName: 'disabled_product',
+        enabled: false,
+      },
+    },
+    {
+      args: {
+        productType: 'without combinations and without available quantities',
+        productToCreate: productWithoutCombinationsWithoutQuantity,
+        gridName: 'no_qty_product_without_combination',
+        enabled: true,
+      },
+    },
+    {
+      args: {
+        productType: 'with combinations and without available quantities',
+        productToCreate: productWithCombinationsWithoutQuantity,
+        gridName: 'no_qty_product_with_combination',
+        enabled: true,
+      },
+    },
+    {
+      args: {
+        productType: 'without price',
+        productToCreate: productWithoutPrice,
+        gridName: 'product_without_price',
+        enabled: true,
+      },
+    },
+  ];
```

Comment:
```suggestion
  ];
  
```

Pleaaase some space in the code, it increase lisibility !

Reply:
done